application-example.yml

```diff
@@ -1,13 +1,24 @@
 rebazer:
+  # Workspace Directory for checkouts, optional.
+  #workspace: ./rebazer-workspace
+
+  # Number of rebases before git garbage collection, optional.
+  #garbageCollectionCountdown: 20
+
+  # Delay between each polling interval, optional.
+  #pollInterval: 60
+  
+  # Regex to match branch names that should be rebased.
+  #branchMatcher: ^feature/.*
```

Comment:
Rename the value to `branchBlacklist`
Default value should be `^(master|develop|release|hotfix)`.